tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "experimentalDecorators": true,
     "sourceMap": true
   },
-  "exclude": [
-    "node_modules"
+  "files": [
+  	"lib/index.ts"
```

Comment:
Can you explain why did you make this change?
Did you test the bundling works properly and we have 2 separate working packages of api/api+ui?